include/k52/optimization/steepest_descent_method.h

```diff
@@ -0,0 +1,120 @@
+#ifndef STEEPEST_DESCENT_METHOD_H_
+#define STEEPEST_DESCENT_METHOD_H_
+
+#include <boost/shared_ptr.hpp>
+
+#include <k52/optimization/continuous_optimizer.h>
+
+namespace k52
+{
+namespace optimization
+{
+
+
+/// @class SteepestDescentMethod
+/// @brief Realization of Steepest descent optimization method.
+class SteepestDescentMethod : public ContinuousOptimizer
+{
+public:
+	/// @brief Creates instance of HookeJeevesMethod.
+	/// @param alfa - number that defines optimal multiplication step for Golden Section Search method
+	/// @param increment_of_the_argument - number that defines value of gradient step per coordinate
+	/// @param max_iteration_number - maximum number of iterations before computations will be stopped
+	/// @param precision - number that defines accuracy of our computations
+	SteepestDescentMethod(
+		double alpha = 1.6,
```

Comment:
If the value is optimal, why make it variable? Or is it just heuristically optimal, not mathematically proved?